### PR TITLE
Update description to manage expectations

### LINKED ISSUE #2555: Manage expectations of Minetest Game
According to paramat, MTG is not a game:

> MTgame is not meant to be a complete game, it is a basic sandbox meant to be modded.

https://forum.minetest.net/viewtopic.php?p=364736#p364736

However, the way how MTG is currently presented to people gives a strong impression that MTG is a real game (instead of a “modding base”).

These official resources are affected:

* README
* Minetest Wiki
* minetest.net homepage
* possibly others?

### PR DESCRIPTION
**Pull request by paramat**
Closes #2555 

Screenshot from 2020-09-13 20-09-03

Screenshot from 2020-09-13 20-09-13

Currently, expectations of MTG are not managed, leading to frequent disappointment, and reviews of MT being based on unmodded MTG.
The description appears in the content tab and content downloader so is critically important.
This also prepares for when MTG is no longer bundled with MT, when users will be directed to browse various games for downloading.
The old description is taken from a 6 year old forum topic written by celeron55.

Removed text from old description:
* I removed the mention of being bundled with MT because hopefully that will not be the case soon, and if it is bundled that will already be obvious.
* I removed 'aims to be' because that can be misinterpreted as meaning the game is not yet as described. If this is not a misinterpretation, describing what a game hopes to be in future is disappointing and pointless, it should be described as it is.  
* I removed the mention of 'lightweight' because although it somewhat is, it has excessive and poorly designed ABM use and so i feel does not deserve to be described as 'good lightweight design' anymore. Those who want a lightweight game should be encouraged to go elsewhere and choose a truly lightweight and better designed game.
* I removed the mention of 'moddable' as that might encourage users to consider this a well-designed base for modding, which it actually is not. This also helps by trying to reduce the modding focus on MTG.
* I removed the mention of being 'fairly playable without mods' as many users do not agree, and this sets expectations too high.

New description:
* The 1st section describes the game and warns there are no NPCs, monsters or animals (i am not counting the static insect decorations) to avoid the common 'where are the mobs?' disappointment.
* The 2nd section makes clear that many users will probably want to add mods, to avoid the common disappointment with the unmodded game. It also makes clear many mods are available as solutions to this.
* The 3rd section informs that MTG is well maintained, to make sure users are made aware of a reliable MT game, because many games may not be particularly well maintained.

Any suggestions for improvements?

### CODE REVIEW

**Comment by SmallJoker** [comment]
Looks good to me, but it would be possible to make use of multi-line setting values to keep it < 80 characters:
```
description = """
First line
Second line
Third line
"""
```

**Comment by sfan5** [comment]
Somehow the "Users will probably want to add mods for satisfactory gameplay." leaves a bad taste in my mouth, I'd prefer if the wording was more in the direction of "modding encouraged".

**Comment by paramat** [comment]
Ok, i assumed this needed to be a single line for some reason. Will add line breaks.
> Users will probably want to add mods for satisfactory gameplay

Yes this could be better worded, but i think it needs to be said, in some way, to avoid disappointment. 'modding encouraged' is very different and loses the important meaning. I will think of a new wording.

**Comment by paramat** [comment]
Updated.
Good input sfan5, i agree the previous wording of that section was a little awkward and distasteful.
Now it has the same meaning but stated in a better way.

SmallJoker, in testing i discovered that any text after a newline is not displayed in the content tab, so this must be one long line.

File: game.conf [hunk 1]
```diff
@@ -1,3 +1,3 @@
 name = Minetest Game
 author = Minetest
-description = Bundled by default with Minetest, and aims to be lightweight, moddable, and fairly playable without mods.
+description = A basic exploration, mining, crafting and building sandbox game with no NPCs, monsters or animals. Minetest Game is usually used with mods added, and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
```
Comment (sfan5) [inline game.conf:3]: (removed a comma)
```suggestion
description = A basic exploration, mining, crafting and building sandbox game with no NPCs, monsters or animals. Minetest Game is usually used with mods added and many mods are available for this game. Reliably maintained by Minetest Engine core developers.
```

Comment (paramat) [reply game.conf:3]: That is fine for me, want me too update it or are you offering to alter on merge?

Comment (sfan5) [reply game.conf]: Oh yeah I can just do so myself...

**Review by sfan5 — APPROVED** [review]

**Review by SmallJoker — APPROVED** [review]

**Pull request merged**